src/Microsoft.AspNetCore.Razor/TagHelpers/HtmlAttributeStructure.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Razor.TagHelpers
+{
+    public enum HtmlAttributeStructure
```

Comment:
When talking offline to @Eilon, I learned the word "structure" isn't helpful here. Also don't see much added value in "HTML" or "Attribute" given tag helper authors only see this type when constructing a `TagHelperAttribute` or reading the property. Suggest something between `AttributeValueQuotingStyle` and `Quoting` or `ValueStyle`.